pkg/utils/backoff.go

```diff
@@ -0,0 +1,135 @@
+package utils
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	DefaultInitialInterval = 500 * time.Millisecond
+	DefaultMultiplier      = 1.5
+	DefaultMaxInterval     = 300 * time.Second
+	DefaultMaxCount        = 30
```

Comment:
I just tested this and 30 count is happening before the max interval. Can we bump this to be inline with new max interval please?